modules/db/src/main/scala/Db.scala

```diff
@@ -68,6 +76,16 @@ object Db:
       val q = f.fragment.query(Codecs.federationSummary)
       postgres.use(_.execute(q)(f.argument))
 
+    def countFederationsSummary: IO[Long] =
+      import skunk.implicits.*
```

Comment:
Is it okay to introduce dependency on `skunk` here? Seems `sql` macros belongs to `Sql` object, but otherwise I just would import it on top of the `Db` object declaration

Reply:
yeah you're right, I was just lazy 😂. Moved `sql` stuff to `Sql` object.